### DISCUSSION TITLE
Add 'People' details

### DISCUSSION THREAD

**Discussion by cheesebo** [photoprism/photoprism#3547] [Q&A] [answered]
Is it possible to add details to the People objects - such as an http link?
I've got some old photos from my parents which I'm hosting for my wider family and it would be great if I could add a link on each person that references their family tree entry.

**Comment by graciousgrey** [comment, marked as answer]
Not yet, but we do have a related feature request:

- https://github.com/photoprism/photoprism/issues/1552

Feel free to add a :+1: there :)

**Reply by cheesebo** [reply]
Thank you! Added a thumbs up.
